recipe/bld.bat

```diff
@@ -1,8 +1,6 @@
-python setup.py install
-if errorlevel 1 exit 1
+set SETUPTOOLS_INSTALL_WINDOWS_SPECIFIC_FILES=0
+set SETUPTOOLS_DISABLE_VERSIONED_EASY_INSTALL_SCRIPT=0
+set DISTRIBUTE_DISABLE_VERSIONED_EASY_INSTALL_SCRIPT=0
```

Comment:
I did not it check on Windows but I believe these flags will do the same as the commands below.